portfolio_website.html

```diff
@@ -17,44 +17,191 @@
-      <li><a
-        href="https://www.linkedin.com/in/kaneez-fatima-0a86601b3"
-        target="_blank"
-        rel="noopener noreferrer "
-        ><i class="fa fa-linkedin icon"></i
-      ></a></li>
-     <li> <a
-        href="https://www.instagram.com/fatima98kaneez."
-        target="_blank"
-        rel="noopener noreferrer"
-        ><i class="fa fa-instagram icon"></i></a></li>
-         <li><a href="Mail:Kaneez98fatima@gmail.com" target="_blank" rel="noopener"
-            ><i class="fa fa-envelope icon"></i
+      <section class="main-section img">
+        <div>
+          <h1>Hey There.</h1>
+            <h1>I'm Kaneez Fatima</h1>
+            <h2>I'am a Software Developer</h2>
+          
+    
+          <p class="para">
+            I can help you build a product, feature or website. Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don't hestiate to contact me.
+          </p>
+    
+          <ul>
+         <li class="inline"><a
+            href="https://github.com/Kaneez98Fatima"
+            target="_blank" rel="noopener noreferrer"><i class="fa fa-github icon"></i></a></li>
+          <li class="inline"><a
+            href="https://www.linkedin.com/in/kaneez-fatima-0a86601b3"
+            target="_blank"
+            rel="noopener noreferrer "
+            ><i class="fa fa-linkedin icon"></i
           ></a></li>
-        </ul>
-        </div>
-      </section>
-</body>
-</html>
\ No newline at end of file
+         <li class="inline"> <a
+            href="https://www.instagram.com/fatima98kaneez."
+            target="_blank"
+            rel="noopener noreferrer"
+            ><i class="fa fa-instagram icon"></i></a></li>
+             <li class="inline"><a href="Mail:Kaneez98fatima@gmail.com" target="_blank" rel="noopener"
+                ><i class="fa fa-envelope icon"></i
+              ></a></li>
+            </ul>
+            </div>
+        </section>
+        <section class="work-area">
+          <h4 class="recent-work">My Recent Works </h4>
+          <div class="works">
+            <div class="work-img"></div>
+            <div class="cap">
+              <h3 class="H3-W">Multi-Post Stories</h3>
+              <p class="para-1">
+                  A daily selection of privately personalized reads; no accounts or
+                  sign-ups required. has been the industry's standard dummy text ever
+                  since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+              </div>
+              <div class="flex btn-flex">
+                <button type="button" class="cursor work-btn">css</button>
+                <button type="button" class="cursor work-btn">html</button>
+                <button type="button" class="cursor work-btn">bootstrap</button>
+                <button type="button" class="cursor work-btn">Ruby</button>
+              </div>
```

Comment:
I suggest you wrap this list with `ul` tag and each project technology a `li` tag using the right HTML semantic tag is best practice 👍